### PR TITLE
Fullscreen sidebar

### LINKED ISSUE #117: Re-merge full-screen schedule
I had to revert the merge commit that merged `fullscreen-schedule` into `dev` because there were still issues with the implementation. Specifically, the schedule text itself is inconsistently formatted. @huynchu, could you fix the spacing and formatting in the schedule text and then open a new pull request?

### LINKED ISSUE #118: App Clip Code isn't displayed properly
Currently, the App Clip Code doesn't display properly in full-screen mode, at least in the latest version of Safari.

### PR DESCRIPTION
**Pull request by huynchu**
Old view (archived)
Closes #117 Closes #118

### CODE REVIEW

**Comment by huynchu** [comment]
Updated view with Android QR code
image

File: src/components/Fullscrn_qrcode.vue [hunk 1]
```diff
@@ -3,14 +3,17 @@
     class="mt-3"
     :class="[{ 'bubble-dark': isDarkMode }, { 'bubble-light': !isDarkMode }]"
   >
-    <h3 :class="{ 'text-white': isDarkMode }">Get our mobile app!</h3>
-
-    <div>
-      <p :class="{ 'text-white': isDarkMode }">
-        IOS APP <span :class="{ 'text-white': isDarkMode }"> Andriod APP </span>
-      </p>
+    <h3 :class="{ 'text-white': isDarkMode }">Get The Mobile App!</h3>
```
Comment (Gerzer) [inline src/components/Fullscrn_qrcode.vue:6]: The "T" in "The" should be lowercase (*i.e.*, "t").

File: src/components/Fullscrn_qrcode.vue [hunk 2]
```diff
@@ -3,14 +3,17 @@
...
   >
-    <h3 :class="{ 'text-white': isDarkMode }">Get our mobile app!</h3>
-
-    <div>
-      <p :class="{ 'text-white': isDarkMode }">
-        IOS APP <span :class="{ 'text-white': isDarkMode }"> Andriod APP </span>
-      </p>
+    <h3 :class="{ 'text-white': isDarkMode }">Get The Mobile App!</h3>
+    <div class="box">
+      <p :class="{ 'text-white': isDarkMode }">IOS</p>
```
Comment (Gerzer) [inline src/components/Fullscrn_qrcode.vue:8]: The "I" in "IOS" should be lowercase—*i.e.*, it should be "iOS"

File: src/components/Fullscrn_qrcode.vue [hunk 3]
```diff
@@ -3,14 +3,17 @@
...
       <div>
-        <img src="../../public/app_clip_shuttle.png" alt="ios qr code" />
+        <img src="../../public/App_Clip_Code.svg" alt="ios qr code" />
+      </div>
+    </div>
+    <div class="box">
```
Comment (Gerzer) [inline src/components/Fullscrn_qrcode.vue:13]: There should be more space between the bottom of the App Clip Code and the top of the "Android" text.

File: src/components/Tracker.vue [hunk 4]
```diff
@@ -3,21 +3,54 @@
...
             </b-badge>
           </div>
-          <div id="schedule" class="position-absolute">
-             <Fullscrn_Schedule v-if="fullscreen"></Fullscrn_Schedule>
-          </div>
-          <div id="qrcode" class="position-absolute">
-            <Fullscrn_qrcode v-if="fullscreen"></Fullscrn_qrcode>
+          <div v-if="fullscreen" id="sidebar" class="position-absolute">
+            <div v-if="fullscreen" id="serverStatus">
+              <img
```
Comment (Gerzer) [inline src/components/Tracker.vue:29]: There should be more vertical space above this image.

**Review by Gerzer — CHANGES_REQUESTED** [review]

File: src/components/Tracker.vue [hunk 5]
```diff
@@ -3,21 +3,54 @@
...
+              {{ fullscreen ? "Exit" : "Enter" }} Fullscreen
             </b-badge>
           </div>
-          <div id="schedule" class="position-absolute">
-             <Fullscrn_Schedule v-if="fullscreen"></Fullscrn_Schedule>
-          </div>
-          <div id="qrcode" class="position-absolute">
-            <Fullscrn_qrcode v-if="fullscreen"></Fullscrn_qrcode>
+          <div v-if="fullscreen" id="sidebar" class="position-absolute">
+            <div v-if="fullscreen" id="serverStatus">
```
Comment (Isaac-the-Man) [inline src/components/Tracker.vue:28]: Duplicate id

File: src/components/Fullscrn_qrcode.vue [hunk 6]
```diff
@@ -50,16 +53,17 @@ img {
...
 }
+.box {
+  text-align: center;
+}
 
 .bubble-light {
-  background-color: rgb(235, 235, 235);
-  border-color: rgb(235, 235, 235);
-  opacity: 0.8;
+  background-color: rgb(235, 235, 235, 0.75);
```
Comment (Isaac-the-Man) [inline src/components/Fullscrn_qrcode.vue:61]: You probably mean `rgba()`. Same goes for the following lines.

File: src/components/Tracker.vue [hunk 7]
```diff
@@ -3,21 +3,54 @@
...
+              <img
+                id="logo"
+                src="../../public/android-chrome-192x192.png"
```
Comment (Isaac-the-Man) [inline src/components/Tracker.vue:31]: Use `logo.png` instead to get rid of the white background.

File: src/components/Fullscrn_Schedule.vue [hunk 8]
```diff
@@ -6,8 +6,8 @@
     <h3 :class="{ 'text-white': isDarkMode }">Shuttle Schedule</h3>
```
Comment (Isaac-the-Man) [inline src/components/Fullscrn_Schedule.vue:6]: Re-use the already existing `Schedule.vue` to avoid duplicate components.

**Review by Isaac-the-Man — CHANGES_REQUESTED** [review]
Nice work but some minor changes are needed.

File: src/components/Fullscrn_qrcode.vue (hunk 1 above)
Comment (huynchu) [reply src/components/Fullscrn_qrcode.vue]: fixed with 06650cb563424e310cfa3d362f0139ca70253197.

File: src/components/Fullscrn_qrcode.vue (hunk 6 above)
Comment (huynchu) [reply src/components/Fullscrn_qrcode.vue]: fixed with 06650cb563424e310cfa3d362f0139ca70253197.

File: src/components/Tracker.vue (hunk 7 above)
Comment (huynchu) [reply src/components/Tracker.vue]: fixed with 5c49b8b72e9c565a1273038ad59a50dd63f57b0e.

File: src/components/Tracker.vue (hunk 4 above)
Comment (huynchu) [reply src/components/Tracker.vue]: fixed with 5c49b8b72e9c565a1273038ad59a50dd63f57b0e.

File: src/components/Fullscrn_Schedule.vue (hunk 8 above)
Comment (huynchu) [reply src/components/Fullscrn_Schedule.vue:6]: I'll open a new issue for this.

File: src/components/Tracker.vue (hunk 5 above)
Comment (huynchu) [reply src/components/Tracker.vue]: should be fixed in 5c49b8b72e9c565a1273038ad59a50dd63f57b0e. If youre refering to duplicate id of "serverStatus".

File: src/components/Fullscrn_qrcode.vue (hunk 3 above)
Comment (huynchu) [reply src/components/Fullscrn_qrcode.vue:13]: fixed in 06650cb563424e310cfa3d362f0139ca70253197.

**Comment by huynchu** [comment]
Updated view after fixing latest requests.
image

**Review by Isaac-the-Man — APPROVED** [review]

**Pull request merged**